### ISSUE TITLE
New version and Brazilian Portuguese translation

### ISSUE THREAD

**Issue by antropolouco** [PhiTux/DailyTxT#53]
Hello there!

How are you?

First, thanks for the project, I found it some days ago and it's nice to know it and see it's being upgraded to a new version.

That said, I would like first to ask if ETA for the new version (before March 2025) is still the same, and if there's any way I can help with the new version. I'm asking that also because I'm willing to contribute at least to a translation to pt-BR, but it will make more sense to do so already for the new version, as the current will become obsolete shortly.

Best regards

**Comment by PhiTux** [comment]
Hi, thank you for the warm words!

Yes, the ETA (which says NOT before March 2025) is still valid. I hope, that I can publish it in March or April. In December, I have hardly any time, mainly thanks to [Advent of Code ](https://adventofcode.com) 😆
Right now, I don't need any help. Perhaps, the new version could be "tested" before officially releasing it... 

Yes, a translation will only make sense for the new version! I'll come back to this ticket, when the translation files are ready. Thank you for this contribution :)

**Comment by antropolouco** [comment]
> Hi, thank you for the warm words!
> 
> Yes, the ETA (which says NOT before March 2025) is still valid. I hope, that I can publish it in March or April. In December, I have hardly any time, mainly thanks to [Advent of Code ](https://adventofcode.com) 😆 Right now, I don't need any help. Perhaps, the new version could be "tested" before officially releasing it...

Nice to hear,  best of luck in AoC!

Thanks for sharing, and sorry for not remembering the "NOT" (my bad, my wait). Is the data from latest public version compatible with new version, will be there an easy (or planned) migration from the current version? If so, I can install the last official release and, as soon as you tell me, I could help testing this part too, perhaps.

> Yes, a translation will only make sense for the new version! I'll come back to this ticket, when the translation files are ready. Thank you for this contribution :)

Sure, thank you again. And as same as I just said, just let me know when the translation files are ready, then.

Best regards, abrazos desde Brasil

**Comment by PhiTux** [comment]
As written in the readme, the new version will definitely be compatible with the current one! 
I'm not yet completely sure, how the migration will be implemented, but the changes in the data structure are minimal, there's probably mainly a need to re-encrypt all the data to a new encryption. I hope that I can implement it in such a way, that it just works fully automatically, and you just have to switch the docker images from version 1 to 2. That's at least the plan :)

**Comment by antropolouco** [comment]
Nice, thanks!

I'll let you know when I have my own instance with the current official version, then I will be available for any tests towards the next version.

Saludos

**Comment by antropolouco** [comment]
Well, I'm done already:

DailyTXT v1.0.15 (+ Apache proxy) up and running.

Good night!

**Comment by antropolouco** [comment]
BTW, in the current version it already shows correctly (automatically?) the full date in pt-BR ("segunda-feira", Monday, "dezembro"...), although the calendar on the left isn't translated at all.

![image](https://github.com/user-attachments/assets/743f533b-896c-4fc6-99b1-a5fe04bcd95c)

**Comment by PhiTux** [comment]
No, the date-string is automatically built by your browser, which uses your system-language.

**Comment by antropolouco** [comment]
> No, the date-string is automatically built by your browser, which uses your system-language.

I imagined something like that, hehehe.

Thanks, see you soon.
